extensions/default/src/commandsModule.ts

```diff
@@ -243,33 +207,9 @@ const commandsModule = ({
           `${activeStudyUID || hpInfo.activeStudyUID}:activeDisplaySet:0`
         ];
         stateSyncService.store(stateSyncReduce);
-        // This is a default action applied
```

Comment:
This is just so much cleaner to have the toolbar defined by the commands and state changes it makes.